### PR TITLE
Enhance the Getting Started Guide with the new deletion feature

### PR DESCRIPTION
**Pull request by tobru**
Now that the deletion feature has landed in master of Lieutenant Operator, let's make use of it in the getting started guide.

### CODE REVIEW

**Review by srueg — APPROVED** [review]

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -404,9 +413,23 @@ curl -k -H "${LIEUTENANT_AUTH}" -H "Content-Type: application/json-patch+json" -
...
-
-...
+. Delete the `Cluster` object
++
+[source,shell]
+----
+kubectl -n lieutenant delete cluster ${CLUSTER_ID}
+----
+This will also delete the associated `GitRepo` object and with that the cluster configuration file in the tenant configuration repository as well as the cluster catalog Git repository on GitLab.
+. Delete the `Tenant` object
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/getting-started.adoc:424]:
```suggestion

. Delete the `Tenant` object
```

File: docs/modules/ROOT/pages/getting-started.adoc [hunk 2]
```diff
@@ -404,9 +413,23 @@ curl -k -H "${LIEUTENANT_AUTH}" -H "Content-Type: application/json-patch+json" -
...
++
+[source,shell]
+----
+kubectl -n lieutenant delete tenant ${TENANT_ID}
+----
+This will also delete the associated `GitRepo` object and with that the tenant configuration Git repository on GitLab.
+. Delete the k3d cluster
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/getting-started.adoc:432]:
```suggestion

. Delete the k3d cluster
```

**Review by corvus-ch — APPROVED** [review]
LGTM. The two suggestions I made, add some white space. I find it more readable when looking at the source that way. Take it or leave it. I can live with both.

**Comment by akosma** [comment]
LGTM!

**Pull request merged**